packages/venia-ui/lib/components/CreateAccount/__tests__/createAccount.spec.js

```diff
@@ -67,34 +65,3 @@ test('attaches the submit handler', () => {
 
     expect(typeof onSubmit).toBe('function');
 });
-
-test('calls onSubmit if validation passes', async () => {
```

Comment:
This function was testing `informed` functionality. Instead of adding a new formApiRef to this component I just deleted this test.